From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 10 May 2010 18:51:27 -0400
Subject: [net] niu: compat header cleanup
Message-id: <20100510184911.2781.97820.sendpatchset@prarit.bos.redhat.com>
Patchwork-id: 24971
O-Subject: [RHEL5 PATCH 11/27] compat.h cleanup: niu driver changes
Bugzilla: 546740
RH-Acked-by: Jarod Wilson <jarod@redhat.com>

>From 9284ae18c3612761266891ac19b7990204e9431a Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 10 May 2010 09:35:17 -0400
Subject: [PATCH] niu driver changes

Resolves 546740.

diff --git a/drivers/net/niu.c b/drivers/net/niu.c
index 4106368..e815632 100644
--- a/drivers/net/niu.c
+++ b/drivers/net/niu.c
@@ -30,7 +30,6 @@
 #endif
 
 #include "niu.h"
-#include "niu_compat.h"
 
 #define DRV_MODULE_NAME		"niu"
 #define PFX DRV_MODULE_NAME	": "
@@ -49,19 +48,6 @@ MODULE_VERSION(DRV_MODULE_VERSION);
 #define DMA_44BIT_MASK	0x00000fffffffffffULL
 #endif
 
-#ifndef readq
-static u64 readq(void __iomem *reg)
-{
-	return ((u64) readl(reg)) | (((u64) readl(reg + 4UL)) << 32);
-}
-
-static void writeq(u64 val, void __iomem *reg)
-{
-	writel(val & 0xffffffff, reg);
-	writel(val >> 32, reg + 0x4UL);
-}
-#endif
-
 static struct pci_device_id niu_pci_tbl[] = {
 	{PCI_DEVICE(PCI_VENDOR_ID_SUN, 0xabcd)},
 	{}
diff --git a/drivers/net/niu_compat.h b/drivers/net/niu_compat.h
deleted file mode 100644
index 7fe0496..0000000
--- a/drivers/net/niu_compat.h
+++ /dev/null
@@ -1,6 +0,0 @@
-#ifndef __NIU_COMPAT_H__
-#define __NIU_COMPAT_H__
-
-#define ETH_FCS_LEN               4
-
-#endif
